lighthouse-core/scripts/pptr-run-devtools.js

```diff
@@ -56,6 +56,54 @@ const argv = /** @type {Awaited<typeof argv_>} */ (argv_);
 /** @type {LH.Config.Json=} */
 const config = argv.config ? JSON.parse(argv.config) : undefined;
 
+/**
+ * @template R [unknown]
```

Comment:
explain why this exists and isnt using puppeteer. basically what you already said in GH comment